### PR TITLE
Prepare Release 10.2.2.3

### LINKED ISSUE #220: Prepare release 10.2.2.3
#### Checklist (minor release):
- [x] Create branch
- [x] Update CHANGELOG.md
- [x] Push this issue branch and create PR for it
- [ ] Merge PR into `dev`
- [ ] Create PR to merge `dev` into `main`;
- [ ] Merge PR into `main`;
- [ ] Git tag `main` with 10.2.2.3;
- [ ] Push the 10.2.2.3 tag on `main`; thus, [.github/workflows/pip.yml] and [.github/workflows/conda.yml] are automatically launched on the 10.2.2.3 tag; whence, a GitHub pre-release is created, the Pip packages are pushed to the pre-release, and the Conda packages are pushed to the default `khiops-dev` channel;
- [ ] Launch [.github/workflows/conda.yml] manually, on the 10.2.2.3 tag, selecting the official `khiops` channel;
- [ ] Copy relevant CHANGELOG.md contents to the 10.2.2.3 GitHub pre-release;
- [ ] Launch [.github/workflows/api-docs.yml] manually on the 10.2.2.3 tag and instruct it to deploy the documentation pages;
- [ ] Transform the 10.2.2.3 GitHub pre-release into a release.

### PR DESCRIPTION
**Pull request by popescu-v**
Closes #220

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,15 @@
   - Example: 10.2.1.4 is the 5th version that supports khiops 10.2.1.
 - Internals: Changes in *Internals* sections are unlikely to be of interest for data scientists.
 
+## 10.2.2.3 - 2024-08-02
+
+### Fixed
+- (`core`) API functions handling of unknown parameters: they now fail.
+- (`sklearn`) Failure mode on SciPy sparse array inputs: it is now compatible with Scikit Learn.
```
Comment (folmos-at-orange) [inline CHANGELOG.md:13]: I think it is not worth put this.

**Review by folmos-at-orange — APPROVED** [review]
Approved with a small change

File: CHANGELOG.md (hunk 1 above)
Comment (folmos-at-orange) [reply CHANGELOG.md:13]: (the sklearn change)

Comment (popescu-v) [reply CHANGELOG.md:13]: OK, I will remove the `sklearn` fix mention.

**Pull request merged**